x-pack/docs/en/security/securing-communications/update-tls-certificates.asciidoc

```diff
@@ -0,0 +1,529 @@
+[[update-node-certs]]
+== Updating node security certificates
+If your cluster certificates expired or you have a new Certificate Authority
```

Comment:
The best practice would be to change these (soon) before the certificates expire, can we amend the wording to signal that ?

Reply:
It's not very common for users and organizations to have a new Certificate Authority. Most commonly users want to add a new node and realize that they don't still have the CA key for the CA that they originally used to create the certificates

Reply:
I think we can change this to read:

> If your cluster certificates are expiring soon or you no longer have the key for your existing Certificate Authority (CA), then you'll need to update the certificates for both the transport and HTTP layers.

Reply:
The " or you no longer have the key for your existing Certificate Authority (CA), " is not enough on its own right? The reasons to change certs can be: 

- node certificates are expiring soon
- they need to add new nodes
- they need to rotate node certificates for other reason ( a node key was leaked etc )
- they have a new CA that they need to trust ( because the old one is not in use any more or because the CA key was compromised, etc )

For the first 3 there is a case where they still have access to the original CA that generated the previous certificates and they can us that and a case where they don't so they need to generate/use a new one.

Reply:
I'm going to rewrite most of this section to expand on these use cases.